modules/planning/tasks/deciders/open_space_fallback_decider.cc

```diff
@@ -33,7 +33,7 @@ OpenSpaceFallbackDecider::OpenSpaceFallbackDecider(const TaskConfig& config)
 
 Status OpenSpaceFallbackDecider::Process(Frame* frame) {
   std::vector<std::vector<common::math::Box2d>> predicted_bounding_rectangles;
-  double collision_distance;
```

Comment:
Load this one from configs

Reply:
> Load this one from configs

I change the varaible's name, it is the distance between obstacle and vehicle

Reply:
> > Load this one from configs
> 
> I change the varaible's name, it is the distance between obstacle and vehicle

It is a variable mistake about  `obstacle_to_vichcle_distance` => `obstacle_to_vehicle_distance` ? or other expressions of vehicle?

Reply:
@mickeyouyou Thanks for the catch and I submitted a commit to fix it.

Reply:
> @mickeyouyou Thanks for the catch and I submitted a commit to fix it.

👍